### PR TITLE
Update for BigInt

### PR DESCRIPTION
**Pull request by ksh8281**

### CODE REVIEW

**Review by bbrto21 — APPROVED** [review]
LGTM

File: src/runtime/BigInt.cpp [hunk 1]
```diff
@@ -63,16 +66,35 @@ BigIntData::BigIntData(VMInstance* vmInstance, const char* buf, size_t length, i
...
+    for (size_t i = 0; i < length; i++) {
+        if (UNLIKELY(buf[i] == 'e' || buf[i] == 'E' || buf[i] == '.')) {
+            bf_set_nan(&m_data);
+            return;
+        }
+        if (UNLIKELY((buf[i] == 'x' || buf[i] == 'X') && buf[0] == '-')) {
+            bf_set_nan(&m_data);
+            return;
+        }
+        newBuf[i] = buf[length];
```
Comment (clover2123) [inline src/runtime/BigInt.cpp:85]: This line seems unnecessary.

File: src/runtime/BigInt.cpp [hunk 2]
```diff
@@ -31,7 +31,9 @@ BigIntData::~BigIntData()
 
 BigIntData::BigIntData(BigIntData&& src)
 {
+    bf_init(src.m_data.ctx, &m_data);
     bf_move(&m_data, &src.m_data);
+    bf_init(m_data.ctx, &src.m_data);
```
Comment (clover2123) [inline src/runtime/BigInt.cpp:36]: I'm not sure why the first `bf_init` should be invoked before `bf_move`.
Does `m_data` needs to be initialized before move operation?

File: tools/test/test262/template.xml [hunk 3]
```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <excludeList>
     <!-- exclude each test file -->
+    <test id="built-ins/decodeURI/S15.1.3.1_A2.5_T1"><reason>TOO SLOW</reason></test>
+    <test id="built-ins/decodeURI/S15.1.3.1_A2.5_T1"><reason>TOO SLOW</reason></test>
+    <test id="built-ins/decodeURIComponent/S15.1.3.2_A2.5_T1"><reason>TOO SLOW</reason></test>
+    <test id="built-ins/decodeURIComponent/S15.1.3.2_A2.5_T1"><reason>TOO SLOW</reason></test>
```
Comment (clover2123) [inline tools/test/test262/template.xml:7]: Our test262 driver supports fast/slow mode.
In slow mode, it runs **slow** TCs marked as `SLOW` not `TOO SLOW`.
Currently, Jenkins CI executes test262 with slow mode in `release-64bit-test262` stage.
(You can also find mode option at [here](https://github.com/Samsung/escargot/blob/9f8b6fdb04deecc30b23a2a35c284f0aa64779ba/tools/run-tests.py#L184))

File: src/runtime/BigInt.cpp (hunk 2 above)
Comment (ksh8281) [reply src/runtime/BigInt.cpp:36]: bf_move function is not `move` contents of bf_t...
bf_move function just copy bf_t's content so we have to reset src content

File: src/runtime/BigInt.cpp (hunk 1 above)
Comment (ksh8281) [reply src/runtime/BigInt.cpp:85]: you are right

File: tools/test/test262/template.xml (hunk 3 above)
Comment (ksh8281) [reply tools/test/test262/template.xml:7]: I am going to rename from TOO SLOW to SLOW

**Comment by ksh8281** [comment]
I updated my PR

File: src/runtime/BigInt.cpp [hunk 4]
```diff
@@ -31,7 +31,9 @@ BigIntData::~BigIntData()
 
 BigIntData::BigIntData(BigIntData&& src)
```
Comment (clover2123) [inline src/runtime/BigInt.cpp:32]: One more thing about this constructor.
It seems that this constructor is nowhere used.
Would you check once more?

Comment (ksh8281) [reply src/runtime/BigInt.cpp:32]: You are right. it is used by nowhere. but I want to keep it
because moving bf_t data is little trickly

Comment (clover2123) [reply src/runtime/BigInt.cpp:32]: Oh, I see.

**Review by clover2123 — APPROVED** [review]
LGTM

**Pull request merged**